package.json

```diff
@@ -49,5 +49,8 @@
   },
   "resolutions": {
     "set-value": "^2.0.1"
+  },
+  "dependencies": {
+    "jq": "^1.7.2"
```

Comment:
I don’t understand where all the dependencies are coming from, because you didn’t invent anything new, as far I can see …
Here and the others below … 👇🏼

Any ideas? @Blargian @Mogge

Reply:
@Tirokk this one @Mogge said I would need when he helped me get the dev environment setup. It's required for implementing multilanguage support or for testing related to that. 😃

Reply:
Hey @Mogge , isn’t this needed for the webapp only? And not for the main folder … 🤔

Reply:
It runs over the `scripts´ folder in the main folder. So I guess it is right there. We have to see what happens when running `yarn locales`.